code/modules/clothing/suits/armor.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/suit/armor
 	allowed = null
-	body_parts_covered = CHEST
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
```

Comment:
This var is used for more then just hitting someone protection, this smells wrong to me
What's the plan here exactly?

Reply:
i standardized all armor to cover all limbs except head because of a bunch of really wacky shit with targeting for ranged weapons and melee in terms of how you're protected from damage that resulted in armor either rng just not working for you, or being statistically likely to be completely useless(see: bulletproof armor and the >4 tile projectile deviation meaning 90% of bullets hit limbs instead of chest)